packages/pages/src/dev/server/server.ts

```diff
@@ -51,7 +51,7 @@ export const createServer = async (
       projectStructure.config.envVarConfig.envVarPrefix
     ),
     optimizeDeps: {
-      include: ["react-dom", "react-dom/client"],
+      include: ["react-dom"],
```

Comment:
The client was required for react 18 even though I know it throws a warning. Did this actually change anything for you other than the console warning?

Reply:
It just cleared the console warning for me. I'll change it back.